app/models/apple/episode.rb

```diff
@@ -199,6 +245,12 @@ def apple?
       feeder_episode.apple?
     end
 
+    def apple_json
+      return nil unless api_response.present?
+
+      apple_data
+    end
```

Comment:
Shoehorned in the existing api data accessors into the `Apple::ApiResponse` call structure -- part of integrating the `api_response` attribute.